<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPrinterForm</class>
 <widget class="QWidget" name="QPrinterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_printer_psd">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_printer_bsd">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
